phpunit.xml.dist

```diff
@@ -36,6 +36,9 @@
 					<element key="slowThreshold">
 						<integer>150</integer>
 					</element>
+					<element key="reportLength">
+						<integer>100</integer>
+					</element>
```

Comment:
50/50 on this: as the slowest tests are shown on a trimmed report I am wondering if 20 would be enough to allow any changes to focus on the slowest tests without the potential for a wall of text.

Reply:
This is left over from testing. I'll revert it.